.github/workflows/vercel-preview-shortener.yaml

```
@@ -1,74 +1,75 @@
-            echo "Newer commit found on $CURRENT_BRANCH branch. Skipping build."
-            echo "skip_build=true" >> $GITHUB_OUTPUT
+    - name: Check if shortener should be built
+      id: check-shortener
+      run: |
+        # Force deploy if triggered manually with force_deploy=true
+        if [ "${{ github.event_name }}" = "workflow_dispatch" ] && [ "${{ inputs.force_deploy }}" = "true" ]; then
+          echo "shortener_detected=true" >> $GITHUB_OUTPUT
+          echo "✅ Force deployment requested via workflow_dispatch"
+          exit 0
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)